Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -728,17 +728,10 @@ protected override ReferenceContainerNode CreateReferenceContainerNode()
             return new CommonReferenceContainerNode(this);
         }
 
-        private void FolderChangedOnDisk(object sender, FolderChangedEventArgs e)
+        private void FileChangedOnDisk(object sender, FileChangedOnDiskEventArgs e)
```

Comment:
>FileChangedOnDisk [](start = 21, length = 17)

Ben had some problems recently where moving a folder didn't produce file-changed events for the files in the folder.  Does this API behave more reasonably?